### PR TITLE
Bilous s

### PR DESCRIPTION
**Pull request by BelousSofiya**

### CODE REVIEW

File: BilousS/HW5/task2.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from random import randint
```
Comment (kolyasalubov) [inline BilousS/HW5/task2.py:1]: good job

File: BilousS/HW5/task3.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+fibonacci_range = int(input("Indicate the lenth of Fibonacci line"))
```
Comment (kolyasalubov) [inline BilousS/HW5/task3.py:1]: good job

File: BilousS/HW6/task1.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+even_2 = [i for i in range(1, 11) if not i%2]
```
Comment (kolyasalubov) [inline BilousS/HW6/task1.py:1]: good job

File: BilousS/HW6/task2.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+password = input("Write a password")
```
Comment (kolyasalubov) [inline BilousS/HW6/task2.py:1]: good job

File: BilousS/HW7/Jenny's_secret/Are You Playing Banjo.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+def are_you_playing_banjo(name):
```
Comment (kolyasalubov) [inline BilousS/HW7/Jenny's_secret/Are You Playing Banjo.py:1]: good job

File: BilousS/HW7/Jenny's_secret/Convert a Number to a String!.py [hunk 6]
```diff
@@ -0,0 +1,3 @@
+def number_to_string(num):
```
Comment (kolyasalubov) [inline BilousS/HW7/Jenny's_secret/Convert a Number to a String!.py:1]: good job

File: BilousS/HW7/Jenny's_secret/Convert boolean values to strings 'Yes' or 'No'.py [hunk 7]
```diff
@@ -0,0 +1,3 @@
+def bool_to_word(boolean):
```
Comment (kolyasalubov) [inline BilousS/HW7/Jenny's_secret/Convert boolean values to strings 'Yes' or 'No'.py:1]: good job

File: BilousS/HW7/Jenny's_secret/Counting sheep.py [hunk 8]
```diff
@@ -0,0 +1,2 @@
+def count_sheeps(sheep):
```
Comment (kolyasalubov) [inline BilousS/HW7/Jenny's_secret/Counting sheep.py:1]: good job

File: BilousS/HW7/Jenny's_secret/Find The Distance Between Two Points.py [hunk 9]
```diff
@@ -0,0 +1,2 @@
+def distance(x1, y1, x2, y2):
```
Comment (kolyasalubov) [inline BilousS/HW7/Jenny's_secret/Find The Distance Between Two Points.py:1]: good job

File: BilousS/HW7/Jenny's_secret/Is this my tail.py [hunk 10]
```diff
@@ -0,0 +1,6 @@
+def correct_tail(body, tail):
```
Comment (kolyasalubov) [inline BilousS/HW7/Jenny's_secret/Is this my tail.py:1]: good job

File: BilousS/HW7/Jenny's_secret/Jenny's secret massage.py [hunk 11]
```diff
@@ -0,0 +1,3 @@
+def greet(name):
```
Comment (kolyasalubov) [inline BilousS/HW7/Jenny's_secret/Jenny's secret massage.py:1]: good job

File: BilousS/HW7/Jenny's_secret/Multiples of 3 or 5.py [hunk 12]
```diff
@@ -0,0 +1,6 @@
+def solution(number):
```
Comment (kolyasalubov) [inline BilousS/HW7/Jenny's_secret/Multiples of 3 or 5.py:1]: good job

File: BilousS/HW7/Jenny's_secret/No yelling!.py [hunk 13]
```diff
@@ -0,0 +1,3 @@
+def filter_words(st):
```
Comment (kolyasalubov) [inline BilousS/HW7/Jenny's_secret/No yelling!.py:1]: good job

File: BilousS/HW7/Jenny's_secret/Reverse List Order.py [hunk 14]
```diff
@@ -0,0 +1,2 @@
+def reverse_list(l):
```
Comment (kolyasalubov) [inline BilousS/HW7/Jenny's_secret/Reverse List Order.py:1]: good job

File: BilousS/HW7/Jenny's_secret/Reversing Words in a String.py [hunk 15]
```diff
@@ -0,0 +1,5 @@
+def reverse(st):
```
Comment (kolyasalubov) [inline BilousS/HW7/Jenny's_secret/Reversing Words in a String.py:1]: good job

File: BilousS/HW7/Jenny's_secret/Will_you_make_it.py [hunk 16]
```diff
@@ -0,0 +1,3 @@
+def zero_fuel(distance_to_pump, mpg, fuel_left):
```
Comment (kolyasalubov) [inline BilousS/HW7/Jenny's_secret/Will_you_make_it.py:1]: good job

File: BilousS/HW7/task1.py [hunk 17]
```diff
@@ -0,0 +1,5 @@
+def return_max(num1, num2):
```
Comment (kolyasalubov) [inline BilousS/HW7/task1.py:1]: good job

File: BilousS/HW7/task3.py [hunk 18]
```diff
@@ -0,0 +1,15 @@
+def calculate_char(word):
```
Comment (kolyasalubov) [inline BilousS/HW7/task3.py:1]: good job

File: BilousS/HW8/calc_areas/calc_area.py [hunk 19]
```diff
@@ -0,0 +1,16 @@
+from math import pi
```
Comment (kolyasalubov) [inline BilousS/HW8/calc_areas/calc_area.py:1]: good job

File: BilousS/HW8/calc_areas/for_user.py [hunk 20]
```diff
@@ -0,0 +1,17 @@
+from calc_area import*
```
Comment (kolyasalubov) [inline BilousS/HW8/calc_areas/for_user.py:1]: good job

File: BilousS/HW8/calculator/calculator.py [hunk 21]
```diff
@@ -0,0 +1,29 @@
+from decimal import Decimal
```
Comment (kolyasalubov) [inline BilousS/HW8/calculator/calculator.py:1]: good job

File: BilousS/HW8/calculator/for_user.py [hunk 22]
```diff
@@ -0,0 +1,17 @@
+from calculator import*
```
Comment (kolyasalubov) [inline BilousS/HW8/calculator/for_user.py:1]: good job

File: BilousS/HW8/task1/main.py [hunk 23]
```diff
@@ -0,0 +1,4 @@
+from utils import *
```
Comment (kolyasalubov) [inline BilousS/HW8/task1/main.py:1]: good job

File: BilousS/HW8/task1/models/__init__.py [hunk 24]
```diff
@@ -0,0 +1,4 @@
+from .admin import *
```
Comment (kolyasalubov) [inline BilousS/HW8/task1/models/__init__.py:1]: good job

File: BilousS/HW8/task1/models/admin.py [hunk 25]
```diff
@@ -0,0 +1,7 @@
+#__all__ = ['create_admin']
```
Comment (kolyasalubov) [inline BilousS/HW8/task1/models/admin.py:1]: good job

File: BilousS/HW8/task1/models/user.py [hunk 26]
```diff
@@ -0,0 +1,4 @@
+#__all__ = ['create_user']
```
Comment (kolyasalubov) [inline BilousS/HW8/task1/models/user.py:1]: good job

File: BilousS/HW8/task1/utils/__init__.py [hunk 27]
```diff
@@ -0,0 +1,6 @@
+from .formatter import *
```
Comment (kolyasalubov) [inline BilousS/HW8/task1/utils/__init__.py:1]: good job

File: BilousS/HW8/task1/utils/formatter.py [hunk 28]
```diff
@@ -0,0 +1,4 @@
+#__all__ = ['format_string']
```
Comment (kolyasalubov) [inline BilousS/HW8/task1/utils/formatter.py:1]: good job

File: BilousS/HW8/task1/utils/logger.py [hunk 29]
```diff
@@ -0,0 +1,7 @@
+#__all__ = ['log_in_file']
```
Comment (kolyasalubov) [inline BilousS/HW8/task1/utils/logger.py:1]: good job

File: BilousS/HW8/task2.py [hunk 30]
```diff
@@ -0,0 +1,28 @@
+password = input("Write your password ")
```
Comment (kolyasalubov) [inline BilousS/HW8/task2.py:1]: good job

File: BilousS/HW9/Rect_pygame.py [hunk 31]
```diff
@@ -0,0 +1,56 @@
...
+
+while run:
+    pygame.time.delay(20) # 100 - занадто повільно:(
+    
+    for event in pygame.event.get(): 
+        if event.type == pygame.QUIT:
+            run = False
+
+    keys = pygame.key.get_pressed()
+
```
Comment (kolyasalubov) [inline BilousS/HW9/Rect_pygame.py:36]: good job

File: BilousS/HW9/Tk_OWM.py [hunk 32]
```diff
@@ -0,0 +1,67 @@
+import tkinter as tk
```
Comment (kolyasalubov) [inline BilousS/HW9/Tk_OWM.py:1]: good job

File: BilousS/HW9/task1unit9/cat_question.py [hunk 33]
```diff
@@ -0,0 +1,125 @@
+ # Rules:
+# The cat has a secret number in range 0-100
+# You have to guess this number
+# You have 10 attempts
+# The cat will help you (you will see his clue in the purple rect in the left corner)
+# Print your number (you will see it in the blue rect in the right corner) and
+# press space-button to put it in program
+
```
Comment (kolyasalubov) [inline BilousS/HW9/task1unit9/cat_question.py:8]: excellent job!!!!!!!!!!

File: BilousS/HW1/task1.py [hunk 34]
```diff
@@ -0,0 +1,4 @@
+name = input("What is your name?")
```
[the remark this answers is not in the document]
Comment (BelousSofiya) [reply BilousS/HW1/task1.py:1]: +

File: BilousS/HW1/task2.py [hunk 35]
```diff
@@ -0,0 +1,9 @@
+a = int(input("Write your number, variable a ="))
```
[the remark this answers is not in the document]
Comment (BelousSofiya) [reply BilousS/HW1/task2.py:1]: Thanks

File: BilousS/HW3/task1.py [hunk 36]
```diff
@@ -0,0 +1,58 @@
...
+Errors should never pass silently.
+Unless explicitly silenced.
+In the face of ambiguity, refuse the temptation to guess.
+There should be one-- and preferably only one --obvious way to do it.
+Although that way may not be obvious at first unless you're Dutch.
+Now is better than never.
+Although never is often better than *right* now.
+If the implementation is hard to explain, it's a bad idea.
+If the implementation is easy to explain, it may be a good idea.
+Namespaces are one honking great idea -- let's do more of those!"""
```
[the remark this answers is not in the document]
Comment (BelousSofiya) [reply BilousS/HW3/task1.py:56]: Thanks

File: BilousS/HW3/task2.py [hunk 37]
```diff
@@ -0,0 +1,8 @@
+num = input()
```
[the remark this answers is not in the document]
Comment (BelousSofiya) [reply BilousS/HW3/task2.py:1]: Thanks

File: BilousS/HW3/task3.py [hunk 38]
```diff
@@ -0,0 +1,4 @@
+a = int(input("Write first number: a = "))
```
[the remark this answers is not in the document]
Comment (BelousSofiya) [reply BilousS/HW3/task3.py:1]: Thanks

**Pull request merged**